clients/drcachesim/tracer/syscall_pt_trace.cpp

```diff
@@ -150,12 +152,11 @@ syscall_pt_trace_t::stop_syscall_pt_trace()
                                 pttracer_output_buffer_.data) != DRPTTRACER_SUCCESS) {
         return false;
     }
-
+    recorded_syscall_count_++;
     if (!trace_data_dump(pttracer_output_buffer_)) {
         return false;
     }
     cur_recording_sysnum_ = -1;
-    recorded_syscall_count_++;
```

Comment:
Better to start recorded_syscall_count_ at zero. We can then keep this increment and also increment before returning false.

Reply:
Done.